modules/Sources/Dependencies/ZcashSDKEnvironment/ZcashSDKEnvironmentInterface.swift

```diff
@@ -18,12 +18,13 @@ extension DependencyValues {
 
 extension ZcashSDKEnvironment {
     public enum ZcashSDKConstants {
-        static let endpointTestnetAddress = "testnet.lightwalletd.com"
+        static let endpointTestnetAddress = "lightwalletd.testnet.electriccoin.co"
```

Comment:
Reverting to this for now until we can figure out what's up with testnet.lightwalletd.com